workers/unity/Packages/com.improbable.gdk.tools/Plugins/PluginCompatibilitySetting.cs

```diff
@@ -0,0 +1,89 @@
+using System.Collections.Generic;
+using System.IO;
+using UnityEditor;
+
+namespace Improbable.Gdk.Tools
+{
+    internal enum PluginType
+    {
+        Core,
+        Sdk,
+    }
+
+    internal enum CPUType
+    {
+        ARMv7,
+        Arm64,
+        X86_64,
+        X86,
+        Unused,
```

Comment:
it means we don't care about it in the plugin importer because for some plugins we have the following path convention:
plugin_type/platform/cpu_type

and for others we use
plugin_Type/paltform

Reply:
the other option would be to make it the same for all plugins and always have the cpu in the path

Reply:
I'd prefer it was called `Agnostic` as all the other options refer to specific CPU architectures

Reply:
sure, will change